ExtensiveRoleCheck.py

```diff
@@ -43,6 +43,10 @@ def add_result(self, name, value):
     def _generate(self):
         for entity in self._json_file['items']:
             role_name = entity['metadata']['name']
+            try:
+                iterator = iter(entity['rules'])
+            except TypeError:
+                continue

```

Comment:
Something that's a bit more pythonic (IMO) would be to do something like this:
```suggestion
            if entity['rules']:
```

Then you'd have to move the following lines under the `if`: https://github.com/cyberark/kubernetes-rbac-audit/blob/d933394528d6b99135a90d7b1b14104a421831cb/ExtensiveRoleCheck.py#L50-L61

That way you're just checking to see if there are any results (since `None` is falsy), and it'll skip over the iterating if there are none.